pkg/virt-handler/vm.go

```diff
@@ -1938,6 +1943,16 @@ func getVMIPodNetwork(vmi *v1.VirtualMachineInstance) *v1.Network {
 	return podNetwork
 }
 
+func (d *VirtualMachineController) checkSRIOVInterfacesForMigration(vmi *v1.VirtualMachineInstance) error {
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
+		if iface.SRIOV != nil && !d.clusterConfig.SRIOVLiveMigrationEnabled() {
+			return fmt.Errorf("SRIOVLiveMigration feature-gate is closed cannot migrate VMI with SRIOV interfaces")
```

Comment:
nit: After "closed" add ",".
Also, instead of "cannot" use "can not" or "can't".

Reply:
Done